cmd/stork/stork.go

```diff
@@ -73,6 +73,7 @@ const (
 	storkVersion               = "version"
 	cmName                     = "stork-version"
 	eventComponentName         = "stork"
+	storkNamespaceEnv          = "STORK-NAMESPACE"
```

Comment:
This const is present in [pkg/webhook](https://github.com/libopenstorage/stork/blob/c53ae4bce821a71a3249fbde0ffc68cd68a50c50/pkg/webhookadmission/utils.go#L29) . Let's define it only in one place within Stork.

Reply:
Removed